tests/linear_model/test_glm.py

```diff
@@ -42,6 +45,11 @@ def get_params(self, deep=True):
 X, y = make_classification(chunks=50)
 
 
+@pytest.fixture()
+def glm_regression_data():
```

Comment:
I don't think this is ever used?

Reply:
Thanks for pointing that out. I have now removed it.